src/lib/components/footer.svelte

```diff
@@ -29,6 +29,17 @@
           aria-label="Github"><i class="fa-brands fa-github fa-xl" title="Github" /></a
         >
       </li>
+      <li>
+  <a
+    data-test-id="linkedin-btn"
+    target="_blank"
+    rel="noreferrer"
+    class="transition duration-200 hover:text-[#0e76a8] active:text-primary-100 dark:hover:text-[#fafafa]"
```

Comment:
We can also remove `dark:hover:text....` as we don't need a dedicated style for dark theme (unlike the github one with low contrast).